File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        my_set = set(nums)
        dicty = {}
        for num in my_set:
            dicty[num] = nums.count(num)
        
        dicty_sorted = sorted(dicty, key = dicty.get, reverse = True)
        return(dicty_sorted[:k])

        

File: Data Structures & Algorithms/remove-element/submission-0.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int indexToFill = 0; 
        int indexParsed = 0;
        

        while(indexToFill < nums.length && indexParsed < nums.length){
            if(nums[indexParsed] != val){
                nums[indexToFill] = nums[indexParsed]; 
                indexToFill += 1; 
                indexParsed += 1; 
            }
            else{ 
                indexParsed += 1; 
            }
        }

        return(indexToFill); 

        
    }
}